### DISCUSSION TITLE
Alternative meeting day / time

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1982] [General]
We had some discussion on an alternative day time for our monthly meeting come up here: https://github.com/projectmesa/mesa/discussions/1935

Pinging regular folks - @projectmesa/maintainers 
Did I miss anyone in that list? 

Saturday at the current time was identified as difficult, because it breaks up the weekend for some. 

I will say that the early mornings in EST are difficult for me because of the young kiddos - they often wake up at 6-7am and immediately look for me. Also our current time is not conducive to anyone that is west of the east coast of the U.S. (eg CST, MT, PST) ... not that we currently have a regular contributor, but we have had some folks interested but never made it because of the time.

Q1: How about we throw out some days / times and then maybe do some sort of vote? 
Q2: I believe our end goal is to alternate months -- one month with Saturday at the currrent time and another month with some other day at some other time?

**Comment by jackiekazil** [comment]
Also - we could say -- that whatever we come up with is tentatively the plan for 2024, then we reassess for 2025

**Comment by EwoutH** [comment]
- Q1 - All weekdays. I've created a poll here, could everyone fill that in? Ignore the exact dates, just select the times in the week you're generally well available: https://www.when2meet.com/?23231948-rKO2t (please select the right timezone!)
- Q2 - agreed, alternating is good. Maybe we can also up the frequency to once every three weeks or something. If you can only make half the meetings that's still every six weeks, instead of two full months.

**Reply by EwoutH** [reply]
Thanks for leaving your availability rht and Jackie!

@tpike3 @Corvince @wang-boyu @ankitk50 @quaquel could you also select the times that you're best available during the week?
